### ISSUE TITLE
Missing labels for the search boxes in the readthedocs theme (accessibility issue)

### ISSUE THREAD

**Issue by dmundra** [mkdocs/mkdocs#3045]
We use mkdocs for our company handbook and when running the Wave accessibility evaluation tool we noticed that the search boxes are missing labels. You can see the error here https://wave.webaim.org/report#/https://handbook.civicactions.com/en/latest/search.html?q=leave and screenshot: 

![Wave error report](https://user-images.githubusercontent.com/7096681/202749449-61d0362d-795a-420d-af49-ecb4245a75b2.png)

Recommendation is to use `aria-label` to add the missing label. See https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html for examples. I will submit a pull request to add the missing label.

I reported the same error for the Sphinx readthedocs theme: https://github.com/readthedocs/sphinx_rtd_theme/issues/1379
